site/deno.jsonc

```
@@ -0,0 +1,39 @@
+{
+  "lock": true,
+  "nodeModulesDir": "auto",
+  "fmt": {
+    "proseWrap": "preserve"
+  },
+  "tasks": {
+    "setup": "deno install --allow-scripts && deno task docs:index-versions && deno task docs:genapi",
+    "docs:genapi": "cd api && deno task genapi",
+    "docs:index-versions": "deno -N docs/.vitepress/plugins/current-versions/build-index.ts > docs/.vitepress/plugins/current-versions/index.json",
+    "docs:dev": "deno -A npm:vitepress dev docs",
+    "docs:host": "deno -A npm:vitepress dev docs --host",
+    "docs:build": "deno task docs:vitepress && deno task docs:sitemap",
+    "docs:serve": "deno -A npm:vitepress serve docs",
+    "docs:preview": "deno task docs:build && deno task docs:serve",
+    "docs:vitepress": "deno -A --v8-flags=--max-old-space-size=8192 npm:vitepress build docs",
+    "docs:lint": "deno -ERS npm:markdownlint-cli2 \"./docs/**/*.md\" \"#./docs/ref\"",
```

Add `--allow-scripts` to get rid of the warning (https://github.com/grammyjs/website/actions/runs/11675581115/job/32510393046)
```
Warning The following packages contained npm lifecycle scripts (preinstall/install/postinstall) that were not executed:
┠─ npm:@parcel/watcher@2.5.0
┠─ npm:vue-demi@0.14.10
┃
┠─ This may cause the packages to not work correctly.
┖─ To run lifecycle scripts, use the `--allow-scripts` flag with `deno install`:
   deno install --allow-scripts=npm:@parcel/watcher@2.5.0,npm:vue-demi@0.14.10
markdownlint-cli2 v0.14.0 (markdownlint v0.35.0)
Finding: ./docs/**/*.md !./docs/ref
Linting: 408 file(s)
Summary: 0 error(s)
```

```suggestion
    "docs:lint": "deno -ERS --allow-scripts npm:markdownlint-cli2 \"./docs/**/*.md\" \"#./docs/ref\"",
```

---

Fixed in https://github.com/grammyjs/website/pull/1145/commits/db625eb15e8042126decc809955468751e0fb4b3